### PR TITLE
About page

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done
**Don't merge** this PR currently includes placeholder images, and the copy doc is still being finalised

Added /about page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8041/about or visit the [demo](https://juju-is-194.demos.haus/about)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that it matches the [design](https://app.zeplin.io/project/606203889af57e12287b5308/screen/6079637868f7b589d49d8259) and the [copy doc](https://docs.google.com/document/d/186zQ7bYE-jXqNNTyKEvSzjq-yP-MiMW2zHKbgyrBrm4/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/2006

### CODE REVIEW

**Comment by sowasred2012** [comment]
@spencerbygraves here's the demo link - could you take a look and let me know what you think? I tweaked the column layout (after chatting with @lyubomir-popov) to be 5 col + 1 col gap + 6 col, it seemed to help with the Technology tabs section jumping around, but happy to go back to what we had yesterday if you prefer.

**Comment by spencerbygraves** [comment]
Thanks @sowasred2012, I think the columns update is fine. Just a few things I noticed...

- The selected state on both the tabs and side nav aren't working properly. I have to click a second time, away from the area to get the highlight bar to appear
- When switching from the "Charmed Operator Lifecycle Manager" tab to another and back, the image position changes and the strip gets taller/shorter
- There's a rogue text character pushing the Practices strip down
image
- Could we have a bit more space under the Practices title please, same as Technology above

Cheers.

**Comment by sowasred2012** [comment]
@spencerbygraves thanks for that - I've addressed those points, and added the first images.

**Comment by solazio** [comment]
@sowasred2012 @danielmutis  do we have a TY page for t-shirt form submission? Or at least a message to say it was sent successfully?

**Comment by sowasred2012** [comment]
> @sowasred2012 @danielmutis do we have a TY page for t-shirt form submission? Or at least a message to say it was sent successfully?

@solazio @danielmutis that's coming separately in this issue https://github.com/canonical-web-and-design/snap-squad/issues/2008 - working on it right now

File: static/js/tabs.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  setActiveTab(tabs);
+}
+
+/**
+  Attaches a number of events that each trigger
+  the reveal of the chosen tab content
+  @param {Array} tabs an array of tabs within a container
+*/
+function attachEvents(tabs) {
+  tabs.forEach(function (tab) {
```
Comment (solazio) [inline static/js/tabs.js:23]: This code is not transpiled, so `forEach` will not work in older browsers. You should also convert the dom elements list to an array with `[].slice.call()` or similar.

File: static/sass/_patterns_strips.scss [hunk 2]
```diff
@@ -231,3 +231,19 @@
...
+
+@mixin jaas-p-strip-blue {
+  .p-strip--blue {
+    @extend %vf-strip;
+    background-color: #00365c;
+
+    &.is-dark {
+      color: $color-x-light;
+
+      .link--light {
```
Comment (solazio) [inline static/sass/_patterns_strips.scss:243]: I guess you could use `.p-link--inverted` instead

File: templates/about/_practices_tabs.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      <h3 class="u-hide--large">Excellent communities build excellent user experience.</h3>
+      <p class="p-heading--4">Share operations and integration code</p>
+      
+      <p>Instead of each business writing and maintaining their own scripts to manage their infrastructure, why not share operations and integrations code, through a system that works across substrates, across platforms, across operating systems and through the applications built upon them. Imagine a world where you don’t need to tweak every script, chart, or file to get your scenario to deploy, scale, or upgrade properly - because those operations are handled for you in code, created by yourself and the community around you.</p>
+      
+      <p>What would this allow you to do? Could you focus on the logic - on the business decisions - that give meaning to those operations? Together, we can combine the knowledge of the open source community into shared, open source applications, which are coded to react to different scenarios, situations, and operations as you’d like them to.</p>
+    </div>
+  </div>
+</div>
```
Comment (solazio) [inline templates/about/_practices_tabs.html:80]: missing empty line at the end of the file

File: templates/about/_technology_tabs.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+          ) | safe
+        }}
+      </div>
+      <div class="col-6 col-start-large-7">
+        <p class="p-heading--4">A system to help you move from configuration management to application management across your hybrid cloud estate - through sharable, reusable, tiny applications called Charmed Operators.</p>
+        <a href="https://juju.is/docs" class="p-button--positive">Get started with the Charmed OLM Docs</a>
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (solazio) [inline templates/about/_technology_tabs.html:101]: missing empty line at the end of the file

File: templates/about/index.html [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    </div>
+  </div>
+</section>
+
+<script src="{{ versioned_static('js/tabs.js') }}"></script>
+<script>
+  initTabs(".js-about-tabs");
+  initEqualHeightTabContent(".js-equal-height-tabs")
+</script>
+{% endblock %}
```
Comment (solazio) [inline templates/about/index.html:104]: missing empty line at the end of the file

**Review by solazio — COMMENTED** [review]
Looks good @sowasred2012 , just a few small comments/suggestions.

File: static/js/tabs.js (hunk 1 above)
Comment (sowasred2012) [reply static/js/tabs.js:23]: > You should also convert the dom elements list to an array with [].slice.call() or similar.

Yep, this already happens on [line 80](https://github.com/canonical-web-and-design/juju.is/pull/194/files#diff-b85bda63b95d30915f3f0e42806a0c674460ae7b8868fd028ca21020bda88b0aR80) (there are a couple of instances of `NodeList`s being cast to `Array`s)

File: static/sass/_patterns_strips.scss (hunk 2 above)
Comment (sowasred2012) [reply static/sass/_patterns_strips.scss]: This actually isn't needed - I'll remove it.

File: templates/about/_technology_tabs.html [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+            height="192",
+            width="192",
+            hi_def=True,
+            loading="auto",
+          ) | safe
+        }}
+      </div>
+      <div class="col-6 col-start-large-7">
+        <p class="p-heading--4">A set of tools to help you write Charmed Operators and to package them as Charms.</p>
+        <a href="#" class="p-button--positive">Start Charming with the Charmed Operator SDK</a>
```
Comment (tbille) [inline templates/about/_technology_tabs.html:35]:
```suggestion
        <a href="/docs/sdk" class="p-button--positive">Start Charming with the Charmed Operator SDK</a>
```

File: templates/about/index.html [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+          width="760",
+          hi_def=True,
+          loading="lazy",
+        ) | safe
+      }}
+      <h3 class="p-heading--4">Send us your details</h3>
+      
+      <form class="marketo-form" action="https://pages.ubuntu.com/index.php/leadCapture/save" method="post" id="mktoForm_1337">
+        <label for="Email" class="mktoLabel">Email address:</label>
+        <input required id="Email" name="Email" maxlength="255" type="email" class="mktoField mktoEmailField mktoRequired" required="required" />
```
Comment (tbille) [inline templates/about/index.html:87]: disable form for now with `Coming soon` value in the input

**Review by spencerbygraves — APPROVED** [review]

**Pull request merged**